waterloop-site/src/components/Hero/Landing.tsx

```diff
@@ -10,41 +10,38 @@ type MyProps = { description: string };
 class heroComponent extends React.Component<MyProps> {
   render() {
     return (
-      <div className="landingContainer">
-        <div className="landingContent">
-          {/* waterloop svg */}
-          <div className="landingWaterloopContainer">
-            <img
-              className="landingWaterloop"
-              src={WaterloopImg}
-              alt="waterloop logo"
-            />
+      // ladningAligner nests background image in another div element to provide proper padding between the image and the bottom.
+      <div className="landingAligner">
+        <div className="landingContainer" style={{backgroundImage: `url(${BuildingsImg})`}}>
+          {/* Use flex: 1 on all 3 child divs to centre the title. */}
+          <div className="landingFiller"></div>
+          <div className="landingContent">
+            {/* waterloop svg */}
+            <div className="landingWaterloopContainer">
+              <img
+                className="landingWaterloop"
+                src={WaterloopImg}
+                alt="waterloop logo"
+              />
+            </div>
+            <div className="landingDescriptionContainer">
+              <h3>{this.props.description}</h3>
+            </div>
+            <Button
+              backgroundColor="yellow"
+              textColor="black"
+              text="JOIN US"
+              onClick={() => window.open("recruitment")}
```

Comment:
Currently opens in a new tab, is this the intended outcome?

Reply:
@jxrin should we keep it in a new tab?